src/panels/config/integrations/ha-config-flow-card.ts

```diff
@@ -71,13 +77,18 @@ export class HaConfigFlowCard extends LitElement {
                   ? "_self"
                   : "_blank"}
               >
-                <mwc-list-item hasMeta>
+                <mwc-list-item
+                  aria-label=${this.hass.localize(
```

Comment:
Why are you adding this here?  The text is already in the DOM.  It's not needed.

Reply:
Because we add this like so in many many places, in exactly the same structure.

I'm not a frontender, so I probably miss a lot of contexts. But as far as I understood, based on history, this label was added in other menus (with the exact same structure) later on because of accessibility.

I'm happy to remove them (less code), but I suggest we remove all other cases from the codebase in that case as well.

Reply:
Yeah I've seen it elsewhere and remove it when I find it.  I don't know the original reason, but I can tell you it does nothing for accessibility.  Text in the DOM is always in the accessibility tree unless programmatically hidden using `display: none;`, `aria-hidden="true"`, etc.  It's safe to remove.

The anchor wrapped around the list item is what causes problems.  That's on my long list.  See #12782.

Reply:
Alright, I pushed a commit that cleans them up! Thanks for the lessons here, that is appreciated!